src/Features/CSharp/Portable/QuickInfo/CSharpSyntacticQuickInfoProvider.cs

```diff
@@ -40,7 +43,7 @@ public CSharpSyntacticQuickInfoProvider()
             }
 
             // Now check if we can find an open brace. 
-            var parent = token.Parent;
+            var parent = token.Parent!;
```

Comment:
I'm actually surprised that this is needed. We would throw on `token.Parent.IsKind` if `token.Parent` is null right?

Reply:
IsKind is an extension method that accepts a null receiver and returns false. 😦